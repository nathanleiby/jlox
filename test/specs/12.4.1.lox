class Hello {}
var h = Hello();

h.x = "hello";
print h.x;
h.x = 55;
print h.x;

// Nested classes
h.y = Hello();
h.y.z = "nested";
print h.y.z;
