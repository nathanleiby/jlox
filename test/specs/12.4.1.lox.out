hello
55
nested
